e2e_tests/tests/streaming/test_client.py

```diff
@@ -0,0 +1,76 @@
+import uuid
+
+import pytest
+
+from determined.common.api import bindings
+from determined.common.streams import wire
+from determined.common.streams._client import LomondStreamWebSocket, ProjectSpec, Stream, Sync
```

Comment:
don't import symbols directly; only import modules.

In this case, `from determined.common import streams` and use `streams.LomondStreamWebSocket`, `streams.ProjectSpec`, etc.